third_party/move/move-compiler-v2/src/pipeline/reference_safety_processor.rs

```diff
@@ -1486,9 +1486,12 @@ impl<'env, 'state> LifetimeAnalysisStep<'env, 'state> {
         );
     }
 
-    /// Process a function call. For now we implement standard Move semantics, where every
-    /// output reference is a child of all input references. Here would be the point where to
-    // evaluate lifetime modifiers in future language versions.
+    /// Process a function call. For now we implement standard Move semantics, where
+    /// 1) every output immutable reference is a child of all input references;
```

Comment:
Could you please check if there is already a test that exercises "immut output is a child of mut input ref", and if not, add one?

Reply:
As far as I am concerned, the test `function_ref_err` covers this case: https://github.com/aptos-labs/aptos-core/blob/80ac6802614bccb3d5d85e4fd997b68f88ca9dae/third_party/move/move-compiler-v2/tests/reference-safety/function_ref_err.move